src/layout/topbar/TopbarRight.tsx

```diff
@@ -0,0 +1,32 @@
+import { Box, Flex } from "@chakra-ui/react";
+import Icon from "../../components/Icon";
```

Comment:
small thing, but I think we can use alias for paths like this
`import ConnectButton from "@/components/ConnectButton";`